config/development.config.php.dist

```diff
@@ -1,11 +1,20 @@
 <?php
 
+/**
+ * Configuration file for the zf-development-tool.
+ *
+ * Do not edit this file, or the tool will not work properly!!!
+ * Instead change the settings in `config/autoload/local.php`.
```

Comment:
What is wrong with editing this file? This file represents configuration that you always want in your development environment.